grakn-engine/src/test/java/ai/grakn/engine/EngineTestHelper.java

```diff
@@ -0,0 +1,68 @@
+package ai.grakn.engine;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * <p>
+ * This class instantiates an engine server-side component once, statically until the JVM exists.
+ * Any state cleanup needs to be done by tests themselves - more code, more work, but more performant
+ * test suite. So all unit tests will share the same components and will be written with that in mind.  
+ * </p>
+ * 
+ * @author borislav
+ *
+ */
+public class EngineTestHelper {
```

Comment:
I have just noticed that this class is inside the engine module. We said that we would be keeping the integration tests in the test module. I imagine the integration tests would need a class very similar if not exactly this. Or am I mistaken? 

I am just wondering how we can minimise the code duplication.